client/src/App.js

```diff
@@ -1,13 +1,18 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
-
+import { BrowserRouter } from 'react-router-dom';
+import NavBar from './components/UI/NavBar';
 import PacmanGame from './components/PacmanGame/PacmanGame';
+import './App.css';
 
 class App extends Component {
```

Comment:
Make the changes in the `App.jsx`. `App.js` has been removed because `JSX` syntax shouldn't be in a `.js` file.